manager/assets/modext/widgets/system/modx.panel.dashboard.js

```diff
@@ -324,7 +324,7 @@ Ext.extend(MODx.window.DashboardWidgetPlace,MODx.Window,{
             fld.markInvalid(_('dashboard_widget_err_placed'));
             return false;
         }
-        var rank = s.getTotalCount();
+        var rank = s.data.length;
```

Comment:
@rtripault I think it should be `s.data.length + 1`, otherwise when I add another widget, it has same rank as the previous.

![screen shot 2015-06-26 at 15 24 56](https://cloud.githubusercontent.com/assets/845220/8378133/876351d6-1c17-11e5-9eed-75127a1413f1.png)